From 651802002a29bb40d17c6293f95fb6fea149629e Mon Sep 17 00:00:00 2001
From: Kent Russell <kent.russell@amd.com>
Date: Tue, 31 Jan 2017 08:53:33 -0500
Subject: [PATCH 1756/2094] drm/amdgpu: Remove amdgpu_powerplay references

Powerplay is no longer optional, so remove references to the
amdgpu_powerplay variable, and the conditions that it was used in

Change-Id: I6ab0533ad5ebe969ab97ac28367d8f3b66d5aa94
Signed-off-by: Kent Russell <kent.russell@amd.com>
---
 drivers/gpu/drm/amd/amdgpu/amdgpu_amdkfd.c | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/drivers/gpu/drm/amd/amdgpu/amdgpu_amdkfd.c b/drivers/gpu/drm/amd/amdgpu/amdgpu_amdkfd.c
index 383218c..50476e9 100644
--- a/drivers/gpu/drm/amd/amdgpu/amdgpu_amdkfd.c
+++ b/drivers/gpu/drm/amd/amdgpu/amdgpu_amdkfd.c
@@ -358,8 +358,7 @@ void get_local_mem_info(struct kgd_dev *kgd,
 			mem_info->local_mem_size_public,
 			mem_info->local_mem_size_private);
 
-	if (amdgpu_powerplay || rdev->pm.funcs->get_mclk)
-		mem_info->mem_clk_max = amdgpu_dpm_get_mclk(rdev, false) / 100;
+	mem_info->mem_clk_max = amdgpu_dpm_get_mclk(rdev, false) / 100;
 }
 
 uint64_t get_gpu_clock_counter(struct kgd_dev *kgd)
@@ -376,10 +375,7 @@ uint32_t get_max_engine_clock_in_mhz(struct kgd_dev *kgd)
 	struct amdgpu_device *rdev = (struct amdgpu_device *)kgd;
 
 	/* The sclk is in quantas of 10kHz */
-	if (amdgpu_powerplay)
 		return amdgpu_dpm_get_sclk(rdev, false) / 100;
-	else
-		return rdev->pm.dpm.dyn_state.max_clock_voltage_on_ac.sclk / 100;
 }
 
 void get_cu_info(struct kgd_dev *kgd, struct kfd_cu_info *cu_info)
-- 
2.7.4

